### PR TITLE
[LogicApp] Az.LoficApp the Track 1 to SDK

### PR DESCRIPTION
**Pull request by lijinpei2008**
## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [ ] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by lijinpei2008** [comment]
Property ChangedTime in swagger was readonly: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json#L6985

Property CreatedTime in swagger was readonly: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json#L5263

Property StartTime in swagger was readonly: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json#L5622

Property EndTime in swagger was readonly: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json#L5628

Property ScheduledTime in swagger was readonly: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json#L6328

Property WaitEndTime in swagger was readonly: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json#L5737

Property LastExecutionTime in swagger was readonly: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json#L5478

Property NextExecutionTime in swagger was readonly: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json#L5484

File: src/LogicApp/LogicApp.Management.Sdk/changed-function-details-20241220155040.csv [hunk 1]
```diff
@@ -0,0 +1,27 @@
+"FileName","CsharpFunction","PowerShellFunction"
```
Comment (VeryEarly) [inline src/LogicApp/LogicApp.Management.Sdk/changed-function-details-20241220155040.csv:1]: please remove this file

File: src/LogicApp/LogicApp.Management.Sdk/function-compare.ps1 [hunk 2]
```diff
@@ -0,0 +1,159 @@
+param(
```
Comment (VeryEarly) [inline src/LogicApp/LogicApp.Management.Sdk/function-compare.ps1:1]: please remove this file

**Comment by lijinpei2008** [comment]
1. Remove the local file
2. The name of the property in PartnerContent.cs was changed from B2B to B2b for consistency with previous versions.

File: src/LogicApp/LogicApp.Management.Sdk/changed-function-details-20241220155040.csv (hunk 1 above)
Comment (lijinpei2008) [reply src/LogicApp/LogicApp.Management.Sdk/changed-function-details-20241220155040.csv]: Thank you for your help, it has now been removed.

File: src/LogicApp/LogicApp.Management.Sdk/function-compare.ps1 (hunk 2 above)
Comment (lijinpei2008) [reply src/LogicApp/LogicApp.Management.Sdk/function-compare.ps1]: Thank you for your help, it has now been removed.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**